apps/src/weblab/WebLab.js

```diff
@@ -64,6 +68,16 @@ WebLab.prototype.init = function (config) {
 
   this.skin = config.skin;
   this.level = config.level;
+  this.hideSource = config.hideSource;
+
+  if (config.share) {
+    $("#codeApp").css({
+      top: "0",
+      left: "0",
+      right: "0",
+      bottom: `${WEBLAB_FOOTER_HEIGHT}px`
+    });
+  }
```

Comment:
This is not pretty, but is simple and direct. We could also create a css class, but the `WEBLAB_FOOTER_HEIGHT` is readily available here. `project.js` does something similar.